lib/rules/max-attributes-per-line.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * @fileoverview Define the number of attributes allows per line
+ * @author Filipa Lacerda
+ */
+'use strict'
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+const utils = require('../utils')
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Define the number of attributes allows per line',
+      category: 'Stylistic Issues',
+      recommended: false
+    },
+    fixable: null,
+    schema: [{
+      type: 'object',
+      properties: {
+        'singleline': {
+          type: 'number',
+          minimum: 1
+        },
+        'multiline': {
+          type: 'object',
+          properties: {
+            'max': {
+              type: 'number',
+              minimum: 1
+            },
+            'allowFirstLine': {
+              type: 'boolean'
+            }
+          }
+        }
+      }
+    }]
+  },
+
+  create: function (context) {
+    const configuration = context.options[0] || { singleline: 3, multiline: { max: 1, allowFirstLine: false }}
```

Comment:
> `const configuration = context.options[0] || { singleline: 3, multiline: { max: 1, allowFirstLine: false }}`

In that case, if somebody write `["error", {"singleline": 1}]` then it's going to throw a TypeError at `const multilineMaximum = configuration.multiline.max`. I'd like to define `parseOptions` function:

```js
function parseOptions(options) {
    const result = { singlelineMax: 3, multilineMax: 1, allowFirstLine: false }

    if (options) {
        if (typeof options.singleline === "number") {
            result.singlelineMax = options.singleline
        } else if (options.singleline && options.singleline.max) {
            result.singlelineMax = options.singleline.max
        }

        if (typeof options.multiline === "number") {
            result.multilineMax = options.multiline
        } else if (options.multiline) {
            if (options.multiline.max) {
                result.multilineMax = options.multiline.max
            }
            if (options.multiline.allowFirstLine) {
                result.allowFirstLine = options.multiline.allowFirstLine
            }
        }
    }

    return result
}
```